### PR TITLE
chore: update behavior for connect api command

### PR DESCRIPTION
**Pull request by blackchoey**

### CODE REVIEW

**Review by xzf0587 — DISMISSED** [review]

File: docs/sdk/teamsfx.conversationbot.md [hunk 1]
```diff
@@ -32,14 +32,16 @@ For command and response, you can register your commands through the constructor
...
+        commands: [ new HelloWorldCommandHandler() ],
+    },
   },
 });
 
 // register through `register*` API
 conversationBot.command.registerCommand(new HelpCommandHandler());
 ```
-For notification, you can enable notification at initialization, then send notifications at any time.
+For notification, you can enable notification at initialization, then send notificaations at any time.
```
Comment (wenytang-ms) [inline docs/sdk/teamsfx.conversationbot.md:44]: notification

File: docs/sdk/teamsfx.md [hunk 2]
```diff
@@ -13,12 +13,12 @@
 |  [CertificateAuthProvider](./teamsfx.certificateauthprovider.md) | <b><i>(BETA)</i></b> Provider that handles Certificate authentication |
 |  [Channel](./teamsfx.channel.md) | <b><i>(BETA)</i></b> A [NotificationTarget](./teamsfx.notificationtarget.md) that represents a team channel. |
 |  [CommandBot](./teamsfx.commandbot.md) | <b><i>(BETA)</i></b> A command bot for receiving commands and sending responses in Teams. |
-|  [ConversationBot](./teamsfx.conversationbot.md) | <b><i>(BETA)</i></b> Provide utilities for bot conversation, including: - handle command and response. - send notification to varies targets (e.g., member, group, channel). |
+|  [ConversationBot](./teamsfx.conversationbot.md) | <b><i>(BETA)</i></b> Provide utilities for bot conversation, including: - handle command and response. - send notification to varies targets (e.g., member, channel, incoming wehbook). |
```
Comment (wenytang-ms) [inline docs/sdk/teamsfx.md:16]: webhook

File: docs/sdk/teamsfx.md [hunk 3]
```diff
@@ -13,12 +13,12 @@
...
 |  [ErrorWithCode](./teamsfx.errorwithcode.md) | <b><i>(BETA)</i></b> Error class with code and message thrown by the SDK. |
 |  [Member](./teamsfx.member.md) | <b><i>(BETA)</i></b> A [NotificationTarget](./teamsfx.notificationtarget.md) that represents a team member. |
 |  [MessageBuilder](./teamsfx.messagebuilder.md) | Provides utility method to build bot message with cards that supported in Teams. |
 |  [MsGraphAuthProvider](./teamsfx.msgraphauthprovider.md) | <b><i>(BETA)</i></b> Microsoft Graph auth provider for Teams Framework |
-|  [NotificationBot](./teamsfx.notificationbot.md) | <b><i>(BETA)</i></b> Provide utilities to send notification to varies targets (e.g., member, group, channel). |
+|  [NotificationBot](./teamsfx.notificationbot.md) | <b><i>(BETA)</i></b> Provide utilities to send notification to varies targets (e.g., member, channel, incoming wehbook). |
```
Comment (wenytang-ms) [inline docs/sdk/teamsfx.md:21]: webhook

File: docs/sdk/teamsfx.notificationbot.md [hunk 4]
```diff
@@ -7,7 +7,7 @@
 > This API is provided as a preview for developers and may change based on feedback that we receive. Do not use this API in a production environment.
 > 
 
-Provide utilities to send notification to varies targets (e.g., member, group, channel).
+Provide utilities to send notification to varies targets (e.g., member, channel, incoming wehbook).
```
Comment (wenytang-ms) [inline docs/sdk/teamsfx.notificationbot.md:10]: webhook

**Review by wenytang-ms — APPROVED** [review]

**Review by wenytang-ms — APPROVED** [review]

**Review by xzf0587 — APPROVED** [review]

**Review by tecton — APPROVED** [review]

**Pull request merged**